### ISSUE TITLE
Dashboard shows background only

### ISSUE THREAD

**Issue by mccarthyryanc** [netdata/netdata#9745]
##### Bug report summary
Nothing displayed from web dashboard except netdata background:

![Selection_011](https://user-images.githubusercontent.com/997120/90191508-e15b7180-dd75-11ea-81b0-1db825cd4c98.png)

Tested in Firefox and Chrome, but here is the js console errors in chrome:

![js-errors](https://user-images.githubusercontent.com/997120/90192106-161bf880-dd77-11ea-9284-afa182ba045e.png)

##### OS / Environment

```
Linux ip-XXX-XX-XX-XXX 5.3.0-1032-aws #34~18.04.2-Ubuntu SMP Fri Jul 24 10:06:28 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=18.04
/etc/lsb-release:DISTRIB_CODENAME=bionic
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="18.04.5 LTS (Bionic Beaver)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 18.04.5 LTS"
/etc/os-release:VERSION_ID="18.04"
/etc/os-release:HOME_URL="https://www.ubuntu.com/"
/etc/os-release:SUPPORT_URL="https://help.ubuntu.com/"
/etc/os-release:BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
/etc/os-release:PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
/etc/os-release:VERSION_CODENAME=bionic
/etc/os-release:UBUNTU_CODENAME=bionic

```

##### Netdata version

Netdata version: `netdata v1.24.0-25-nightly`

##### Component Name

web component

##### Steps To Reproduce

1. normal install
2. open browser dashboard

An uninstall and re install of netdata was tried but got the same behavior in the dashboard. 

##### Expected behavior

Charts and other dashboard elements visible.

**Comment by ilyam8** [comment]
Hi @mccarthyryanc 👋 

Show `HOST:api/v1/info` output

**Comment by mccarthyryanc** [comment]
@ilyam8 , Here is the JSON from that endpoint:
<details>
<summary> HOST:api/v1/info </summary>

```json
{
	"version": "v1.24.0-35-nightly",
	"uid": "b5e926f8-dda9-11ea-99c7-06520b99197e",
	"mirrored_hosts": [
		"ip-XXX-XX-XX-XXX"
	],
	"alarms": {
		"normal": 69,
		"warning": 0,
		"critical": 0
	},
	"os_name": "Ubuntu",
	"os_id": "ubuntu",
	"os_id_like": "debian",
	"os_version": "18.04.5 LTS (Bionic Beaver)",
	"os_version_id": "18.04",
	"os_detection": "/etc/os-release",
	"cores_total": "4",
	"total_disk_space": "0",
	"cpu_freq": "2199000000",
	"ram_total": "16643084288",
	"container_os_name": "none",
	"container_os_id": "none",
	"container_os_id_like": "none",
	"container_os_version": "none",
	"container_os_version_id": "none",
	"kernel_name": "Linux",
	"kernel_version": "5.3.0-1032-aws",
	"architecture": "x86_64",
	"virtualization": "kvm",
	"virt_detection": "systemd-detect-virt",
	"container": "unknown",
	"container_detection": "none",
	"host_labels": {
		"_os_name": "Ubuntu",
		"_os_version": "18.04.5 LTS (Bionic Beaver)",
		"_kernel_version": "5.3.0-1032-aws",
		"_system_cores": "4",
		"_system_cpu_freq": "2199000000",
		"_system_ram_total": "16643084288",
		"_system_disk_space": "0",
		"_architecture": "x86_64",
		"_virtualization": "kvm",
		"_container": "unknown",
		"_container_detection": "none",
		"_virt_detection": "systemd-detect-virt",
		"_is_parent": "false"
	},
	"collectors": [
		{
			"plugin": "proc.plugin",
			"module": "/proc/pressure"
		},
		{
			"plugin": "apps.plugin",
			"module": ""
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/diskstats"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/softirqs"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/uptime"
		},
		{
			"plugin": "cgroups.plugin",
			"module": "systemd"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/meminfo"
		},
		{
			"plugin": "tc.plugin",
			"module": ""
		},
		{
			"plugin": "idlejitter.plugin",
			"module": ""
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/dev"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/loadavg"
		},
		{
			"plugin": "proc.plugin",
			"module": "ipc"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/sockstat"
		},
		{
			"plugin": "statsd.plugin",
			"module": "stats"
		},
		{
			"plugin": "cgroups.plugin",
			"module": "stats"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/interrupts"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/snmp"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/stat"
		},
		{
			"plugin": "netdata",
			"module": "stats"
		},
		{
			"plugin": "diskspace.plugin",
			"module": ""
		},
		{
			"plugin": "web",
			"module": "stats"
		},
		{
			"plugin": "go.d",
			"module": "prometheus"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/netstat"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/rpc/nfs"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/sys/kernel/random/entropy_avail"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/vmstat"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/snmp6"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/softnet_stat"
		},
		{
			"plugin": "proc.plugin",
			"module": "/proc/net/sockstat6"
		},
		{
			"plugin": "ebpf.plugin",
			"module": ""
		}
	],
	"cloud-enabled": true,
	"cloud-available": true,
	"agent-claimed": false,
	"aclk-available": false
}
```

</details>

**Comment by ilyam8** [comment]
i think disabling prometheus collector will do (`go.d.conf`)

https://github.com/netdata/go.d.plugin/blob/5f61c42fdd4e0e4b4801d4ac574cc78a5f9a0dec/config/go.d.conf#L45

**Comment by TheOldMan2000** [comment]
I've experienced this a lot when scrolling or jumping to charts near the bottom of the page, usually they just suddenly disappear. I thought it was an iPad issue.

**Comment by ilyam8** [comment]
> scrolling or jumping to charts near the bottom of the page

@TheOldMan2000 that is another problem. I suggest to record a short video and open an issue.

>  I thought it was an iPad issue.

If i undertand you right the problem exists only on ipad. Am i right?

**Comment by TheOldMan2000** [comment]
Okay, thanks. It just looks the same as the screenshot the OP posted above, all of the graphs just suddenly disappear but it happens randomly. I thought it may be my 128GB device but it has plenty of free memory.

"If i undertand you right the problem exists only on ipad. Am i right?"

I think so but not 100% sure, it reaches 36C in my attic home office with the windows open so using my iPad downstairs mostly at the moment!

**Comment by mccarthyryanc** [comment]
@ilyam8 disabling prometheus worked! For the curious, here is what I did:
```bash
sudo /etc/netdata/edit-config go.d.conf
```
and then modified `#  prometheus: yes` to `  prometheus: no`, and the restart:
```bash
sudo service netdata restart
```

I didn't really think about this before, but this same instance is running GitLab, which comes with prometheus monitoring. Is it possible the two are intefering with each other? Searching through `/etc/netdata/netdata.conf` I noticed some gitlab-related prometheus settings.
